diff --git a/dist/amd-library.js b/dist/amd-library.js
index 49025dc..2720959 100644
--- a/dist/amd-library.js
+++ b/dist/amd-library.js
@@ -736,16 +736,15 @@ define(function () { 'use strict';
 
 	  };
 
-	  if (typeof define === 'function' && define.amd) {
+	  if (typeof exports === 'object') {
+	    // CommonJS
+	    module.exports = EventEmitter;
+	  } else if (typeof define === 'function' && define.amd) {
 	     // AMD. Register as an anonymous module.
 	    define(function() {
 	      return EventEmitter;
 	    });
-	  } else if (typeof exports === 'object') {
-	    // CommonJS
-	    module.exports = EventEmitter;
-	  }
-	  else {
+      } else {
 	    // Browser global.
 	    window.EventEmitter2 = EventEmitter;
 	  }
diff --git a/dist/umd-library.js b/dist/umd-library.js
index 9d39494..bf05fe1 100644
--- a/dist/umd-library.js
+++ b/dist/umd-library.js
@@ -740,16 +740,15 @@
 
 	  };
 
-	  if (typeof define === 'function' && define.amd) {
+	  if (typeof exports === 'object') {
+	    // CommonJS
+	    module.exports = EventEmitter;
+	  } else if (typeof define === 'function' && define.amd) {
 	     // AMD. Register as an anonymous module.
 	    define(function() {
 	      return EventEmitter;
 	    });
-	  } else if (typeof exports === 'object') {
-	    // CommonJS
-	    module.exports = EventEmitter;
-	  }
-	  else {
+      } else {
 	    // Browser global.
 	    window.EventEmitter2 = EventEmitter;
 	  }
